plugin/bufsize/setup.go

```diff
@@ -0,0 +1,56 @@
+package bufsize
+
+import (
+	"fmt"
+	"strconv"
+
+	"github.com/coredns/coredns/core/dnsserver"
+	"github.com/coredns/coredns/plugin"
+
+	"github.com/caddyserver/caddy"
+)
+
+func init() { plugin.Register("bufsize", setup) }
+
+func setup(c *caddy.Controller) error {
+	bufsize, err := bufsizeParse(c)
+	if err != nil {
+		return plugin.Error("bufsize", err)
+	}
+
+	dnsserver.GetConfig(c).AddPlugin(func(next plugin.Handler) plugin.Handler {
+		return Bufsize{Next: next, Size: bufsize}
+	})
+
+	return nil
+}
+
+func bufsizeParse(c *caddy.Controller) (int, error) {
+	for c.Next() {
+		args := c.RemainingArgs()
+		switch len(args) {
+		case 0:
+			// Nothing specified; use 512 as default
+			bufsize := 512
+			return bufsize, nil
```

Comment:
```suggestion
return defaultBufSize, nil
```

Define `const defaultBufSize = 512` and delete line above.